assembly/src/snapshots/miden_assembly__tests__basic_block_and_simple_if_true-2.snap

```diff
@@ -4,11 +4,11 @@ expression: program
 ---
 begin
     join
-        basic_block push(2) push(3) end
+        basic_block push(2) push(3) noop noop noop noop noop noop noop end
```

Comment:
I think this block should have 2 noops.

Reply:
Oh, the *groups* (not batches) are padded to a power of two? That's a bit surprising (though @plafer 's [comment](https://github.com/0xMiden/miden-vm/pull/2092#discussion_r2294213775)) helps, but it can be done.

Reply:
Isn't it 1 NOOP? There's 1 batch, where the groups are

0: the operation group containing `[PUSH, PUSH]`
1: immediate value `2`
2: immediate value `3`
3: (padding) operation group containing `[NOOP]`

And no padding needed for the number of batches.